### PR TITLE
Mrc-5420 Remove config file and inject env variables

### PR DESCRIPTION
**Pull request by absternator**

### CODE REVIEW

**Review by EmmaLRussell — COMMENTED** [review]
Logic looks good to me, but I couldn't log in with the super user. Maybe I copied the values incorrectly from the vault, but it looked ok to me...

**Comment by absternator** [comment]
> Logic looks good to me, but I couldn't log in with the super user. Maybe I copied the values incorrectly from the vault, but it looked ok to me...

Did ya wanna try now I made a script called create-super-user

File: scripts/create-super-user [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+# OFR BASI AUTH ONLY!!!
```
Comment (EmmaLRussell) [inline scripts/create-super-user:4]:
```suggestion
# FOR BASIC AUTH ONLY!!! 
```
??

File: README.md [hunk 2]
```diff
@@ -113,7 +113,4 @@ You will need access to the vault to run the `githubauth` configuration, which r
 details.
 
 ### Notes
-
-1. If running app in **basic auth** mode, a super admin user can be created by running
-`docker exec packit-packit-db create-super-user --email <ADMIN_EMAIL> --password <ADMIN_PASSWORD_ENCODED> --uuid <ADMIN_UUID>`.
-The *< >* fields can all be retrieved from vault at `secret/packit/basicauth`
+1. If running app in **basic auth** mode, a super admin user can be created by running `./scripts/create_superuser.sh` after the app is running.
```
Comment (EmmaLRussell) [inline README.md:116]:
```suggestion
1. If running app in **basic auth** mode, a super admin user can be created by running `./scripts/create-super-user` after the app is running.
```

**Review by EmmaLRussell — COMMENTED** [review]
Thanks, that worked for me! Must have just copied something wrong! :laughing: 

Could you update the version too please? Think you just need to update in `src/packit_deploy/__about__.py`

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**